plugin/completion/lib_complete.py

```diff
@@ -330,6 +330,33 @@ def update(self, view, settings):
         log.error("no translation unit for view id %s", v_id)
         return False
 
+    def get_reference(self, view):
```

Comment:
This is probably now the best name. It should specify a reference of what we are getting.